full-installer.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+echo "Desenvolvido por Guilherme:) www.facebook.com/guilhermexyz"
+echo "----------------------------------------------------------"
+sudo su
+apt-get install python-dev 
+apt-get install python-virtualenv
+virtualenv .
+source bin/activate
+pip install -r requirements.txt
+echo "KUdS API initialized...."
+echo " by guilherme"
+echo "---------------------"
+wget http://pgoapi.com/pgoencrypt.tar.gz
+tar -xf pgoencrypt.tar.gz
+cd pgoencrypt/src/ 
+make
+mv libencrypt.so ../../encrypt.so
+cd ..
+cd ..
+cd configs/
+cp config.json.bak  config.json
+echo "configuring ......"
+echo "----------------------"
+echo "Enter with you ptc login: "
+read ptcLog
+echo "-----------------------"
+echo "Enter with your ptc password: "
+read ptcPass
```

Comment:
consider adding the option `-s` to the `read` command, that will prevent it from displaying it while typing